package.json

```diff
@@ -73,7 +73,7 @@
     "@wordpress/jest-puppeteer-axe": "1.6.0",
     "@wordpress/plugins": "^3.1.0",
     "@wordpress/postcss-themes": "2.3.0",
-    "@wordpress/rich-text": "3.24.1",
+    "@wordpress/rich-text": "^4.1.2",
```

Comment:
Not related to this line: but should we keep version 2 in `package-lock.json` file?